src/3d/qgspointcloudlayerchunkloader_p.cpp

```diff
@@ -181,7 +291,21 @@ void QgsPointCloud3DSymbolHandler::makeEntity( Qt3DCore::QEntity *parent, const
 
   Qt3DRender::QPointSize *pointSize = new Qt3DRender::QPointSize( renderPass );
   pointSize->setSizeMode( Qt3DRender::QPointSize::Programmable );  // supported since OpenGL 3.2
-  pointSize->setValue( mSymbol->pointSize() );
+  switch ( mSymbol->renderingStyle() )
+  {
+    case QgsPointCloud3DSymbol::RenderingStyle::NoRendering:
+      // Do Nothing since there is no rendering
+      break;
+    case QgsPointCloud3DSymbol::RenderingStyle::SingleColor:
+      pointSize->setValue( dynamic_cast<QgsSingleColorPointCloud3DSymbol *>( mSymbol.get() )->pointSize() );
```

Comment:
I'd move pointSize() up to the base QgsPointCloud3DSymbol class to avoid this switch